codecov.yml

```diff
@@ -0,0 +1,17 @@
+codecov:
```

Comment:
yep, if we need `codecov.yml` for coverage to work correctly, we should add it to `webpack-defaults` aswell to keep repos in sync